modules/openlmis-web/src/main/webapp/public/js/distribution/model/adult-coverage.js

```diff
@@ -122,22 +151,22 @@ function AdultCoverageLineItem(lineItem) {
   this.healthCenterTetanus2To5 = this.healthCenterTetanus2To5 || {};
 }
 
-function setNotRecorded(field) {
+function setNotRecorded(field, notRecordedApplied) {
```

Comment:
Because this function sets notRecorded to the opposite of the value passed in by the user, a name like “toggleNotRecorded” may be a little more descriptive.